lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -301,3 +303,126 @@ def upload_ora2_data(
     TASK_LOG.info(u'%s, Task type: %s, Upload complete.', task_info_string, action_name)
 
     return UPDATE_STATUS_SUCCEEDED
+
+
+def upload_students_grades_data(
```

Comment:
Could you move out the logic with grade calculation (gathering grade info, query user, etc) It allows to reduce method size and allows decomposition of the logic.

Reply:
Done